tests/integration/commands/dev/dev.zisi.test.ts

```diff
@@ -19,6 +21,51 @@ const __dirname = path.dirname(fileURLToPath(import.meta.url))
 
 const testMatrix = [{ args: [] }]
 
+const generateSSLCertificate = async () => {
+  const tmpdir = await fs.mkdtemp(path.join(os.tmpdir(), 'certs-'))
```

Comment:
👁️ you look like someone who would enjoy removing the `tempy` dev dep `:grindog:`

Reply:
dev deps are free!